src/Avalonia.Visuals/Platform/IPlatformAccentColorProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Avalonia.Media;
+
+namespace Avalonia.Platform
+{
+    public interface IPlatformAccentColorProvider
```

Comment:
Lets call this IPlatformColorSchemeProvider... as this will need to also provide information about light and dark mode in the future among other things. Like if we are in high contrast mode also.

Reply:
@PieroCastillo

Reply:
Yes, it's not, but we can avoid unnecessary breaking changes in the future.